dico/voice/websocket.py

```diff
@@ -115,8 +115,11 @@ async def process(self, resp: GatewayResponse):
             self.heartbeat_interval = resp.d["heartbeat_interval"]
             self._heartbeat_task = self.client.loop.create_task(self.run_heartbeat())
         elif resp.op == VoiceOpcodes.HEARTBEAT_ACK:
-            self.last_heartbeat_ack = time.time()
-            self.ping = self.last_heartbeat_ack - self._ping_start
+            if(int(self.last_heartbeat_send * 1000) != resp.d):
```

Comment:
```suggestion
            if int(self.last_heartbeat_send * 1000) != resp.d:
```
This is not python style.